markdown/ch5.markdown

```diff
@@ -1175,14 +1175,14 @@ For example, if every GT segment $g$ is split into two equally-sized
 segments $a_1$ and $a_2$ in AS, then $H(AS|GT) = 1$, because after knowing a
 pixel is in $g$, you
 still need 1 additional bit to know whether it belongs to $a_1$ or $a_2$.
-However, H(GT|AS) = 0, because regardless of whether a pixel is in $a_1$ or
+However, $H(GT | AS) = 0$, because regardless of whether a pixel is in $a_1$ or
```

Comment:
No worries, done! I only updated the uppercase GT. There's "gt" used earlier in the chapter.